doc/rados/operations/health-checks.rst

```diff
@@ -824,9 +824,10 @@ Or, to disable this alert on a specific OSD, run the following command:
 BLOCK_DEVICE_STALLED_READ_ALERT
 _______________________________
 
-There are certain BlueStore log messages that surface storage drive issues 
+There are BlueStore log messages that surface storage drive issues 
```

Comment:
I have a suggestion which might lead to a discussion:

s/surface/reveal/

I think that "surface" sounds jargony and "reveal" sounds more direct.